_plugins/to_gravatar.rb

```diff
@@ -0,0 +1,17 @@
+require 'digest/md5'
+
+module Jekyll
+  module ToGravatarFilter
+    def to_gravatar(input, size=100)
+      "//www.gravatar.com/avatar/#{hash(input)}?s=#{size}"
```

Comment:
I'd define also default fall-back image here: `&d=mm` to use "mystery man".

Otherwise we'll see uggly G icons.

http://en.gravatar.com/site/implement/images/